config/peerpods/podvm/libvirt-podvm-image-cm.yaml

```diff
@@ -22,3 +22,6 @@ data:
   ORG_ID: ""
   ACTIVATION_KEY: ""
   BASE_OS_VERSION: "9.2"
+
+  # To Enable SE for IBM Z
+  SE_BOOT: "1"
```

Comment:
Better to use `yes/no` or `true/false` if this is tied to changes in cloud-api-adaptor - https://github.com/confidential-containers/cloud-api-adaptor/commit/d5ca6f3d5997f0ae6112e030254a3776e4403f09